clang/test/CIR/CodeGen/union.c

```diff
@@ -5,25 +5,146 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -emit-llvm %s -o %t.ll
 // RUN: FileCheck --check-prefix=OGCG --input-file=%t.ll %s
 
+union U1 {
```

Comment:
Can you add tests for a union that requires padding, a union that is packed, and a union that isn't zero-initializable?